ietf/community/models.py

```diff
@@ -13,13 +12,23 @@
 from ietf.utils.models import ForeignKey
 
 class CommunityList(models.Model):
-    user = ForeignKey(User, blank=True, null=True)
+    person = ForeignKey(Person, blank=True, null=True)
     group = ForeignKey(Group, blank=True, null=True)
     added_docs = models.ManyToManyField(Document)
 
+    @property
+    def user(self):
+        try:
+            return self.person.user
+        except AttributeError:
+            return None
+
     def long_name(self):
-        if self.user:
-            return 'Personal I-D list of %s' % self.user.username
+        if self.person:
+            try:
+                return 'Personal I-D list of %s' % self.person.user.username
```

Comment:
Lets change this to use self.person.plain_name

Reply:
Okay.